AutoEncoderTest_Stichprobe.py

```diff
@@ -0,0 +1,604 @@
+"""
+Code ist im Rahmen der Bachelorarbeit von Annika Kies am FZI Karlsruhe mit dem Titel
+"Entwicklung und Analyse von Auto-Encodern für intelligente Agenten zum Erlernen von Atari-Spielen"
+entstanden.
+
+Code dient zum Testen einzelner Bilder mit einem trainierten Auto-Encoder.
+Dafür kann manuell ein Bild übergeben werden, oder mehrere Bilder ab einem bestimmten Index im Datensatz.
+
+Implementiert sind vier Auto-Encoder Architekturen.
+AutoEncoder1 orientiert sich an:
+Paper: Ha and Schmidhuber, "World Models", 2018. https://doi.org/10.5281/zenodo.1207631.
+https://github.com/ctallec/world-models
+"""
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from matplotlib import pyplot as plt
+from PIL import Image
+import numpy as np
+from numpy import load
+from sklearn.model_selection import train_test_split
+from torch.utils.data import TensorDataset
+from numpy import asarray
+from visdom import Visdom
+import utils
+import cv2
+
```

Comment:
Hier der selbe Comment wie in den anderen zwei Dateien: Die AutoEncoder nur an einer Stelle implementieren und dann dort wo sie gebraucht werden importieren